addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -518,7 +518,11 @@ class StudyService extends Service {
           .key('category', category)
           .limit(1000)
           .projection(fields)
-          .query();
+          .query()
+          // filter by aws region. Because the scraper might not run until after
+          // this code is deployed we want this filter to be permissive when it isn't
+          // clear if the study is in the correct region
+          .filter(study => ['us-east-1', 'unknown', undefined].includes(study.region))
```

Comment:
I bet there's a place where we can pass in the region we're in (env vars?), instead of hard-coding the `us-east-1` region we want.

Reply:
Good idea.